resolvers/webpack/index.js

```diff
@@ -44,12 +49,16 @@ exports.resolve = function (source, file, settings) {
   }
 
   var webpackConfig
-
-  var configPath = get(settings, 'config')
-    , configIndex = get(settings, 'config-index')
+    , configPath
+    , configIndex
     , packageDir
 
-  log('Config path from settings:', configPath)
+  // Only respect webpackConfig settings if not in a node_module
+  if (!~file.indexOf('/node_modules/')) {
```

Comment:
please never use `!~`, that's far too clever for its own good - explicitly compare to a number instead, since that's what `indexOf` returns.

Reply:
that is a pretty neat combo operator, but I agree with @ljharb 😁